app/models/swanson_quote.rb

```diff
@@ -0,0 +1,15 @@
+class SwansonQuote < ApplicationRecord
+  validates :quote, presence: true
+  has_many :quote_ratings
+
+  def self.rate(quote, user_ip, rating)
+    #rate this quote
+    the_quote = self.find_by(quote: quote)
```

Comment:
If this was an instance variable you wouldn't need to find it. it would just be @swanson_quote.rate params